test/transform/validation_model.jl

```diff
@@ -0,0 +1,81 @@
+
+#
+# Julia implementation of LSTM/RNN classifier for classifying program
+# execution traces. 
+# 
+
+#
+# Loop to check if packages are installed and install them if not, then 
+# import.
+# 
+
+using Pkg
+
+packages = ['Flux','DelimitedFiles','MLDataPattern']
+
+for package in packages
+    haskey(Pkg.installed(),package) || Pkg.add(package)
+
+using DelimitedFiles
+using Flux
+using Flux: onehot, throttle, crossentropy, onehotbatch, params, shuffle
+using MLDataPattern: stratifiedobs
+using Base.Iterators: partition
+
+
+#
+# Set up inputs for model
+#
+
+# Read lines from traces.dat text in to arrays of characters
+# Convert to onehot matrices
+
+cd(@__DIR__)
+
+text = collect.(readdlm("traces.csv"));
+alphabet = [unique(vcat(text...))..., '\n'];
```

Comment:
This seems like a useful function to put in a file like `src/validation/utils.jl`

Reply:
Will do!